<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileShareServer</class>
 <widget class="QMainWindow" name="FileShareServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件共享</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="userTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>文件名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>上传时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>上传用户</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>上传IP</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="UpLoadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownLoadButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
